libut/Makefile

```diff
@@ -1,33 +1,41 @@
-OBJS=libut.a
-all: $(OBJS) 
+SHELL=/usr/sh
+VERSION=2.0.1
+MAJOR=2
```

Comment:
You mean...to use "2.0.1" directly instead of $(VERSION)? then 2.0.1 and 2 will become magic numbers, you'll have to change all of them in this Makefile at version updates. Or you just want to move them down to the place where they are about to be used right away?